static/js/sections.js

```diff
@@ -116,23 +116,38 @@ let atomic_interval = null;
 
 function toggleOperationView(){
     if($('#togBtnOp').is(':checked')) {
-        showHide('#queueName,#queueGroup,#queueFlow,#opBtn,#queueCleanup,#queueStealth,#queueSource','#operations');
+        showHide('.queueOption,#opBtn','#operations');
     } else {
-        showHide('#operations','#queueSource,#queueName,#queueGroup,#queueFlow,#opBtn,#queueCleanup,#queueStealth,#queueCleanup,#queueStealth');
-    }
+        showHide('#operations,#opBtn','.queueOption');
+    } 
 }
 
 function handleStartAction(){
     let name = document.getElementById("queueName").value;
-    if(!name){alert('Please enter an operation name!'); return; }
+    if(!name){alert('Please enter an operation name'); return; }
+
+    let jitter = document.getElementById("queueJitter").value
+    try {
+        let [jitterMin, jitterMax] = jitter.split("/");
+        jitterMin = parseInt(jitterMin);
+        jitterMax = parseInt(jitterMax);
+        if(!jitterMin || !jitterMax || jitterMin >= jitterMax){
+            throw "jitter min greater than jitter max";
```

Comment:
this does not seem like it gets hit..? I tried 6/5 in the jitter field and got the alert from the catch below instead..?